app/resources/create.html.template

```diff
@@ -28,7 +28,7 @@
     {{env.hidden_input_tags_for_preserved_query_params|safe}}
     <input type="hidden" name="role" value="{{params.role}}">
     <div class="create" role="form">
-
+    <div class="column centered">
       <div class="column start">
```

Comment:
I believe you no longer need this "column start" element. Can you try deleting it?

"column start" is used together with "column end" to define a multi-column layout.